src/components/app.js

```diff
@@ -0,0 +1,15 @@
+import Header from "./header";
+import OpenSource from "./open-source";
+import WorkHere from "./work-here";
+
+const App = () => {
+	return (
+		<div>
+			<Header>Social Tables Engineering</Header>
```

Comment:
do you need the `Social Tables Engineering` here if you're setting it via a string in `header.js`?